guess.js

```diff
@@ -0,0 +1,39 @@
+var pry = require('pryjs');
+
+function GuessingGame(answer) {
+  this.answer = answer;
+  this.guess = function(num){
+    if (num === this.answer) {
+      this.solvedStatus = true;
+      return "correct"
+    } else if (num > this.answer) {
+      return "high"
+    } else {
+      return "low"
+    }
+  };
+  this.solvedStatus = false;
+  this.isSolved = function(){
```

Comment:
For functions, look into assigning them to a constructor's prototype object instead of within the constructor.